libs/product/driver/magento/src/queries/fragments/product.ts

```diff
@@ -28,6 +28,18 @@ export const magentoProductFragment = gql`
 		description {
 			html
 		}
+		upsell_products {
+      ...magentoProductPreview
+      ...magentoBundledProduct
+      ...magentoSimpleProduct
+      ...magentoConfigurableProduct
+    }
+    related_products {
+      ...magentoProductPreview
+      ...magentoBundledProduct
+      ...magentoSimpleProduct
+      ...magentoConfigurableProduct
+    }
 		...magentoBundledProduct
 		...magentoSimpleProduct
 		...magentoConfigurableProduct
```

Comment:
You should be able to remove `magentoConfigurableProduct` since you are providing it in the module.

Reply:
I don't want to introduce breaking changes yet, and this would be a breaking change unless you imported the `DaffConfigurableProductMagentoDriverModule`